### ISSUE TITLE
negative binomial doesn't work, but zinb does

### ISSUE THREAD

**Issue by jeff-regier** [YosefLab/scVI#111]
If I create a VAE with "nb" rather than "zinb", I get the error
```
E       RuntimeError: expand(torch.cuda.FloatTensor{[128]}, size=[]): the number of sizes provided (0) must be greater or equal to the number of dimensions in the tensor (1)
```

I added a unit test marked xfail to show the problem:
https://github.com/YosefLab/scVI/blob/40e65e99c84793157de762a4e9421f4c60fac181/tests/test_scvi.py#L211-L219

I think @maxime1310 is using negative binomial for smFISH data---otherwise I'd propose just deleting the code for negative binomial.

**Comment by Edouard360** [comment]
Ah mistake just came from the fact that we summed over the batch dim in nb:

https://github.com/YosefLab/scVI/blob/40e65e99c84793157de762a4e9421f4c60fac181/scvi/metrics/log_likelihood.py#L53

https://github.com/YosefLab/scVI/blob/40e65e99c84793157de762a4e9421f4c60fac181/scvi/metrics/log_likelihood.py#L70

**Comment by Edouard360** [comment]
Included the fix in #112

**Comment by maxime1310** [comment]
I didn't have this bug in my version but thanks for thinking about it @jeff-regier! It would indeed have been quite unpleasant to realize that I had been working with a bug since the beginning
